### ISSUE TITLE
[Task]: Implement basic send form popup for stardust profiles

### ISSUE THREAD

**Issue by nicole-obrien** [iotaledger/firefly#3078]
### Task description

We need to change the send form popup for stardust based profiles, to align with new designs. While we are waiting on all the API changes it is also best to refactor the send functionality that is split across multiple UI files. i.e. the `Send.svelte` is 700 lines long and some functionality is in the `Wallet.svelte` file to.

### Requirements

- [ ] Move send functionality to a popup
- [ ] Create new sub components
- [ ] Refactor app logic for sending
- [ ] Remove old code

### Acceptance criteria

PR approved and merged into `stardust-develop`

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project
